tests/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.Tests/FacebookWrapperTests.cs

```diff
@@ -95,5 +97,71 @@ public async void SendMessageAsyncShouldThrowAnExceptionWithNullPayload()
                 await facebookWrapper.SendMessageAsync("wrongPath", null, null, default(CancellationToken));
             });
         }
+
+        [Fact]
+        public async void VerifyWebhookAsyncShouldSendOKWhenVerified()
+        {
+            var facebookClientWrapper = new FacebookClientWrapper(_testOptions);
+            var httpRequest = new Mock<HttpRequest>();
+            var httpResponse = new Mock<HttpResponse>();
+
+            httpRequest.SetupGet(req => req.Query[It.IsAny<string>()]).Returns("TestVerifyToken");
+
```

Comment:
remove extra space

Reply:
I was living it in because the upper line sets up httpRequest and the lines below set the httpResponse. It could get hard to read with more variables. 
I'll remove the space anyway